.github/pull_request_template.md

```diff
@@ -1,12 +1,11 @@
-**Fix ...**
-
 **Changes:**
 * Briefly describe changes
 
 **I have:**
 <!-- fill in with [x] -->
 - [ ] Set target branch to the correct branch, usually `dev`.
+- [ ] Rebased source branch from the target branch.
 - [ ] Ensured code is formatted, linted.
```

Comment:
"Ensured code is formatted, linted." should be "Ensured code is formatted and linted."